src/components/widgets-settings/hist-mode/hist-mode.tmpl.html

```diff
@@ -1,12 +1,12 @@
 <div class="settings hist-mode-choser">
   <div common-time-period-info context="forwardParams" />
 
-  <div align="center" class="options">
+  <div align="center" class="options" ng-show="!parentWidget.isLiveBalance">
```

Comment:
Wow, this is very intrusive: you should pass a parameter to the directive instead, something like "show-mode-choser" which would be true by default